handout/a11y/key-concerns/keyboard-accessibility.md

```diff
@@ -0,0 +1,61 @@
+# Keyboard Accessibility
+
+Keyboard accessibility is the ability of your application to be interacted with using just a keyboard. The more streamlined the site can be used this way, the more keyboard accessible it is. Keyboard accessibility is one of the largest aspects of web accessibility since it targets:
+
+- those with motor disabilities who can't use a mouse
+- blind users who typically prefer to navigate websites with their keyboard
```

Comment:
Might be good to mention that many different forms of assistive technology rely on an app to be keyboard accessible here.